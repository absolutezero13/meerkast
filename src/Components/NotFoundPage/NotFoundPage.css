.not-found-page {
  display: flex;
  justify-content: center;
  align-items: center;
  padding-bottom: 7%;

  height: 68vh;
}
.not-found-page a {
  color: rgba(1, 126, 254, 255);
}
