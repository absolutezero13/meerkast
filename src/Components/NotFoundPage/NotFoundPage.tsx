import React from "react";
import { Link } from "react-router-dom";
import "./NotFoundPage.css";
const NotFoundPage: React.FC = () => {
  return (
    <div className="not-found-page">
      <h1>
        ⚠️ PAGE NOT FOUND. GO BACK TO <Link to="/meerkast">HOME. ➡️</Link>
      </h1>
    </div>
  );
};

export default NotFoundPage;
