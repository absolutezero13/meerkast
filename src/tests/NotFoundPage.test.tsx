import React from "react";
import { render } from "@testing-library/react";
import NotFoundPage from "../Components/NotFoundPage/NotFoundPage";
import { BrowserRouter } from "react-router-dom";

const { container } = render(
  <BrowserRouter>
    <NotFoundPage />{" "}
  </BrowserRouter>
);

// SnapShot Test
it("Not found page matches snapshot", () => {
  expect(container).toMatchSnapshot();
});
