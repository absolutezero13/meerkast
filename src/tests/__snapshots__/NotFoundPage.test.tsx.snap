// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page matches snapshot 1`] = `
<div>
  <div
    class="not-found-page"
  >
    <h1>
      ⚠️ PAGE NOT FOUND. GO BACK TO 
      <a
        href="/"
      >
        HOME. ➡️
      </a>
    </h1>
  </div>
   
</div>
`;
